express/code/blocks/frictionless-quick-action-mobile/frictionless-quick-action-mobile.js

```diff
@@ -189,7 +188,7 @@ export default async function decorate(block) {
   // TODO: remove fallback row once authoring is done
   const [fallbackRow] = rows.filter((row) => row.children[0]?.textContent?.toLowerCase()?.trim() === 'fallback');
   fallbackRow?.remove();
-  if (fallbackRow && getMobileOperatingSystem() !== 'Android') {
+  if (fallbackRow) {
```

Comment:
we would need to work with authoring team first to make sure they have removed all the fallback row from our pages already, before we can update this. Otherwise all pages will suddenly use the fallback content